app/models/provider.rb

```diff
@@ -75,4 +75,8 @@ def lacks_admin_users?
   def geocoded?
     latitude.present? && longitude.present?
   end
+
+  def vacancies?
+    course_options.any? { |course_option| course_option.vacancy_status == 'vacancies' }
```

Comment:
Should we move the verification `vacancy_status == 'vacancies'` to the course option model?
Can you add unit tests for this method too?

Reply:
The product decission to re-work the page means all this logic is removed.